src/shared/components/feature-length-ruler/FeatureLengthRuler.tsx

```diff
@@ -79,7 +80,15 @@ const FeatureLengthRuler = (props: Props) => {
       <g>
         <rect className={styles.tick} width={1} height={6} />
         <text className={styles.label} x={0} y={20} textAnchor="end">
-          {props.unitsLabel ? `${props.unitsLabel} 1` : 1}
+          {props.rulerLabel && (
+            <tspan
+              className={styles.rulerName}
+              x={`-${props.rulerLabelOffset ?? 2}ch`}
+            >
+              {props.rulerLabel}
+            </tspan>
+          )}
+          <tspan x={0}>1</tspan>
```

Comment:
Does this need the `label` className?

Reply:
Hmm. At the moment, the parent `text` element has the `label` classname, which should apply to this `tspan`. I am not sure whether it makes more sense to move the `label` from the parent to this tspan. It works as it is though.